enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/utils.ts

```diff
@@ -0,0 +1,100 @@
+import dayjs from "dayjs";
+import { c } from "ttag";
+
+import type { StaleCollectionItem } from "metabase-types/api";
+
+export const itemKeyFn = (item: StaleCollectionItem) =>
+  `${item.id}:${item.model}`;
+
+const labelContext = `Time span the item has not been viewed in, reads as "Not viewed in over <time span>"`;
+
+export const dateFilterOptions = [
+  // TODO: remove option before merging
+  {
+    label: c(labelContext).t`-1 days`,
```

Comment:
You probably need to put the actual string into the `c` function for this to work

Reply:
Similar convo to the select one. I'm bringing this up to product, so I don't want to rework this if they want to redesign the input somehow.

Reply:
Fixed i think